deploy/openshift/templates/che-server-template.yaml

```diff
@@ -0,0 +1,262 @@
+    triggers:
+    - type: ConfigChange
+
+parameters:
+- name: ROUTING_SUFFIX
+  displayName: Routing suffix of your OpenShift cluster
+  description: This is the suffix appended to routes in your OpenShift cluster <route>-<project>.<routing_suffix>
+- name: CHE_VERSION
+  displayName: Eclipse Che version
+  description: Eclipse Che version which defaults to nightly
+  value: nightly
+- name: IMAGE_CHE
+  displayName: Eclipse Che server image
+  description: Che server Docker image. Defaults to docker.io/eclipse/che-server
+  value: docker.io/eclipse/che-server
+- name: CHE_MULTIUSER
+  displayName: Single vs Multi User Che
+  description: False ie single user by default. Set to true
+  value: 'false'
+- name: OPENSHIFT_USERNAME
+  displayName: OpenShift Username
+  description: OpenShift username that will be used to create workspace objects
+- name: OPENSHIFT_PASSWORD
+  displayName: OpenShift Password
+  description: OpenShift Password that will be used to create workspace objects
+- name: OPENSHIFT_TOKEN
+  displayName: OpenShift token
+  description: OpenShift token to be used to create workspace objects. Must be set if username/password are impossible to use. Do not set if username/pass are provided!
+- name: PROTOCOL
+  displayName: https or http protocol
+  description: Protocol to be used in Che communications
+  value: http
+- name: WS_PROTOCOL
+  displayName: wss or ws
+  description: Websocket protocol to be used in Che communications. Defaults to ws. Set to wss only if TLS is true and https protocol is defined
+  value: ws
+- name: TLS
+  displayName: HTTPS support
+  description: Turn on https support in Che
+  value: 'false'
+- name: CHE_INFRA_OPENSHIFT_PROJECT
+  displayName: OpenShift namespace to create workspace objects
+  description: Che server namespace by default. When empty, each workspace gets own namespace. In this case a privileged sa or credentials/token need to be used.
+  value: '${NAMESPACE}'
+- name: CHE_INFRA_KUBERNETES_PVC_QUANTITY
+  displayName: Default PVC claim
+  description: Default PVC claim for storage. Defaults to 1Gi
+  value: '1Gi'
+- name: CHE_INFRA_KUBERNETES_PVC_STRATEGY
+  displayName: PVC strategy
+  description: PVC strategy. Unique implies creating PVC per workspace. Common uses one PVC with subpaths in PV. Defaults to unique
+  value: 'unique'
+- name: CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD
+  displayName: Admin password update
+  description: Force an admin to update password after 1st login. True by default
+  value: 'true'
+- name: CHE_KEYCLOAK_AUTH__SERVER__URL
+  displayName: Identity provider URL
+  description: URL of a remote identity provider. Defaults to Keycloak bundled with Che multi user
+  value: '${PROTOCOL}://keycloak-${NAMESPACE}.${ROUTING_SUFFIX}/auth'
+- name: CHE_JDBC_URL
+  displayName: JDBC URL
+  description: URL Che server will use to connect to JDBC. Defaults to Postgres service bundled with Che multi user
+  value: 'jdbc:postgresql://postgres:5432/dbche'
```

Comment:
it seems this URL is for multi user che
the command of README using template with a single user che is failing then as it tries to connect on postgresql
did I miss something ?

Reply:
(single user default url being jdbc:h2:che)

Reply:
Fixed. DB URL can be passed as env not a param, while for default single and multi user uses cases, server has got props